tests/settings.py

```diff
@@ -0,0 +1,21 @@
+# -*- encoding: utf-8 -*-
+from __future__ import unicode_literals
+
+import os
+import tempfile
+
+BASE_DIR = os.path.dirname(os.path.abspath(__file__))
+
+INSTALLED_APPS = (
+    'django.contrib.admin',
+    'django.contrib.auth',
+    'django.contrib.contenttypes',
+    'django.contrib.sessions',
+    'django.contrib.sites',
+    'public_admin',
+    'tests'
+)
+
+DEFAULT_FILE_STORAGE = 'tests.storage.MyFileSystemStorage'
```

Comment:
Do we really need it? I mean, [Django already has a default value to use if this variable is unset](https://docs.djangoproject.com/en/3.0/ref/settings/#std:setting-DEFAULT_FILE_STORAGE) and I don't get why we should set it to an non-existent object ; )

Reply:
Yes, in the end, I added those extra things thinking that they might be useful, but makes sense to remove those lines.